pkg/config/config.go

```diff
@@ -120,6 +120,15 @@ func LoadIPAMConfig(bytes []byte, envArgs string, extraConfigPaths ...string) (*
 		}
 	}
 
+  // Set the final value of old range paramaters too for consistency
+
+	if n.IPAM.Range != "" {
+    idx := len(n.IPAM.IPRanges)-1
+    n.IPAM.Range = n.IPAM.IPRanges[idx].Range
+    n.IPAM.RangeStart = n.IPAM.IPRanges[idx].RangeStart
+    n.IPAM.RangeEnd = n.IPAM.IPRanges[idx].RangeEnd
+	}
```

Comment:
This code is not indented correctly. 

Please run a formatter in your code.

Reply:
Thanks. Done :+1: